transport/tchannel/handler.go

```diff
@@ -195,31 +195,36 @@ func (h handler) callHandler(ctx context.Context, call inboundCall, responseWrit
 		tracer := h.tracer
 		ctx = tchannel.ExtractInboundSpan(ctx, tcall.InboundCall, headers.Items(), tracer)
 	}
-
+	span := opentracing.SpanFromContext(ctx)
```

Comment:
Probably, I'm loosing the idea of changes.

There's a function called `handle` (not `Handle`), why it should not have code like

```
extractOpenTracingSpan := &transport.ExtractOpenTracingSpan{
		ParentSpanContext: parentSpanCtx,
		Tracer:            tracer,
		TransportName:     TransportName,
		StartTime:         start,
		ExtraTags:         yarpc.OpentracingTags,
	}
	ctx, span := extractOpenTracingSpan.Do(ctx, transportRequest)
	defer span.Finish()
```

And I'd expect in this case span should be updated only once - based on the error returned from the `callHandler` func.